packages/nouns-webapp/src/pages/Vote/index.tsx

```diff
@@ -417,7 +410,7 @@ const VotePage = ({
         <Row>
           <Col xl={4} lg={12}>
             <Card className={classes.voteInfoCard}>
-              <Card.Body className="p-2">
+              <Card.Body className="p-2" style={{marginBottom:"0.9rem"}}>
```

Comment:
Since it looks like this project is using Bootstrap, I think you can also use `.mb-3` which will look similar.